### ISSUE TITLE
Add `_CTC_c_under6_bonus`

### ISSUE THREAD

**Issue by MaxGhenis** [PSLmodels/Tax-Calculator#2294]
Two major tax credit bills split out children under age 6:
* [American Family Act](https://www.vox.com/future-perfect/2019/3/6/18249290/child-poverty-american-family-act-sherrod-brown-michael-bennet) (Bennet-Brown child allowance)
* [Working Families Tax Relief Act](https://www.vox.com/future-perfect/2019/4/10/18302183/tax-cut-democrats-earned-income-tax-credit-child-allowance)

There's currently a `_CTC_c_under5_bonus` that can model this, but a `_CTC_c_under6_bonus` would be more accurate.

This depends on https://github.com/PSLmodels/taxdata/issues/311 to add `nu06`.

**Comment by martinholmer** [comment]
Given the discussion in PSLmodels/taxdata#311, it would seen as if the thing to do here is to **replace** the old (almost never used) `CTC_c_under5_bonus` policy parameter with the new `CTC_c_under6_bonus` policy parameter.

**Comment by Thirdhuman** [comment]
Yes, "under 6" is far more useful for modeling policy proposals currently being considered.

**Comment by martinholmer** [comment]
@Thirdhuman said in Tax-Calculator issue #2294:
> Yes, "under 6" is far more useful [than "under 5"] for modeling policy proposals currently being considered.

@andersonfrailey will be replacing the `nu05` variable with a `nu06` variable in both the PUF and CPS input data files.  Once those files are available, I can make the change in the policy parameter to use the new variable.

**Comment by Thirdhuman** [comment]
In the meantime, simply turning all the 5 year olds into 4 year olds in my CPS input file should make `nu05` function as under as if it were `nu06` in a provisional sense, correct?

**Comment by martinholmer** [comment]
@Thirdhuman said in issue #2294:
> In the meantime, simply turning all the 5 year olds into 4 year olds in my CPS input file should make nu05 function as under as if it were nu06 in a provisional sense, correct?

Yes, I guess so, assuming you reconstruct the `nu05` **variable** after you make all the five year olds be four.

**Comment by Thirdhuman** [comment]
@martinholmer - would you mind confirming my understanding here on where this provisional recode is properly implemented?  

This change would be performed after I've performed after I've produced the .csv on the relevant input file. Command:
 `* ./taxcalc.sh [file name] save`

But before I perform the following commands on the resulting csv:
* tc [file name].csv 2017 --dump
* tc [file name].csv 2017 --dump --reform 'policy-change.json'

Is my understanding of when this provisional change can be properly executed without causing difficulties correct?

**Comment by martinholmer** [comment]
@Thirdhuman asked in issue #2294:
> would you mind confirming my understanding here on where this provisional recode [of the input variable `nu05`] is properly implemented?
>
> This change would be performed after I've performed after I've produced the .csv on the relevant input file.
Command:
>```
>* ./taxcalc.sh [file name] save
>```

I don't think that will work because the `taxcalc.sh` script does not generate an input .csv file that includes the `nu05` variable.  The only CPS-based input file that contains the `nu05` variable is the `Tax-Calculator/taxcalc/cps.csv.gz` file.  If you want to somehow recode the `nu05` variable in that file, things might work.  But my recommendation would be to wait until @andersonfrailey replaces the `nu05` variable with a new `nu06` variable in both the `cps.csv.gz` and `puf.csv` files.  We will then issue a new release of Tax-Calculator that uses that variable in CTC reform calculations.

> Is my understanding of when this provisional change can be properly executed without causing difficulties correct?

In a word, NO.  You seem to be a new user of Tax-Calculator and what you're trying to do is pretty advanced.
I'd recommend that you wait for the new version of Tax-Calculator.
